src/server/api/routers/traces.ts

```diff
@@ -275,4 +275,37 @@ export const traceRouter = createTRPCRouter({
         observations: enrichedObservations as ObservationReturnType[],
       };
     }),
+  delete: protectedGetTraceProcedure
```

Comment:
This also allows public access if trace is published (meant for reading only). Use protectedProjectProcedure which requires the projectId as a param